.solcover.js

```diff
@@ -8,7 +8,15 @@ module.exports = {
     shell.rm('-rf', './build/typechain'); // Clean up at the end
   },
   // TODO: remove VirtualSwap from skipFiles once coverage is added
-  skipFiles: ['helper/test/TestSwapReturnValues.sol', 'VirtualSwap/'],
+  // TODO: add coverage for older contracts
+  skipFiles: [
+    'helper/test/TestSwapReturnValues.sol',
+    'VirtualSwap/',
+    'LPTokenGuarded.sol',
+    'OwnerPausable.sol',
+    'SwapGuarded.sol',
+    'SwapUtilsGuarded.sol'
```

Comment:
nit: maybe put all the guard related contracts in a directory together like `VirtualSwap/`